### DISCUSSION TITLE
Naming SVG-Favicon

### DISCUSSION THREAD

**Discussion by astridx** [joomla/joomla-cms#33078] [Q&A] [answered]
Wouldn't it make sense to just call the SVG-File for the Favicon in Cassiopeia as well `favicon.svg` - without `joomla `in front of it. 

https://github.com/joomla/joomla-cms/blob/4.0-dev/build/media_source/system/images/joomla-favicon.svg

Joomla users who overwrite Cassiopeia would not have to name their favicon` joomla-favicon.svg` or edit `index.php` - It is often a logo of the own company or organisation.

Or am I missing something?

**Comment by dgrammatiko** [comment, marked as answer]
> Joomla users who overwrite Cassiopeia would not have to name their favicon joomla-favicon.svg

I didn't realise that the name of the favicon was important, thus the `joomla-` prefix. If you think that this worth fixing please do a PR renaming them

**Comment by sonnemondundsterne** [comment]
I wonder for what we need the mask-icon

https://github.com/joomla/joomla-cms/blob/f4ac18829fad6eadd7260ac2f951aca0c7da459b/templates/cassiopeia/index.php#L25

See: https://twitter.com/RealFavicon/status/1054986646150627329

**Reply by dgrammatiko** [reply]
I just used https://css-tricks.com/svg-favicons-and-all-the-fun-things-we-can-do-with-them/ as my guide. You could remove it (although for me plain svg favicon is still not showing up in the safari tabs)
